portal/apps/org.wso2.carbon.iam.portal.common/pom.xml

```diff
@@ -0,0 +1,115 @@
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2.is</groupId>
+        <artifactId>org.wso2.carbon.iam.portal.parent</artifactId>
+        <version>6.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.carbon.iam.portal.common</artifactId>
+    <packaging>uuf-component</packaging>
+    <name>WSO2 IS UUF common</name>
+    <url>http://wso2.org</url>
+
+    <dependencies>
+        <!--UUF Components-->
+        <dependency>
+            <groupId>org.wso2.carbon.uuf.common</groupId>
+            <artifactId>org.wso2.carbon.uuf.common.foundation.ui</artifactId>
+            <version>${carbon.uuf.common.version}</version>
+            <type>zip</type>
+            <classifier>uuf-component</classifier>
+        </dependency>
+
+        <!--UUF Themes-->
+        <dependency>
+            <groupId>org.wso2.carbon.uuf.common</groupId>
+            <artifactId>org.wso2.carbon.uuf.common.default.theme</artifactId>
+            <version>${carbon.uuf.common.version}</version>
+            <type>zip</type>
+            <classifier>uuf-theme</classifier>
+        </dependency>
+        <!--Backend dependencies-->
+        <dependency>
+            <groupId>org.wso2.is</groupId>
+            <artifactId>org.wso2.is.portal.user.client.api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.wso2.carbon.identity.mgt</groupId>
+            <artifactId>org.wso2.carbon.identity.recovery</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.wso2.carbon.uuf.maven</groupId>
+                <artifactId>carbon-uuf-maven-plugin</artifactId>
+                <version>${carbon-uuf-maven-plugin.version}</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>create-component</goal>
+                        </goals>
+                        <configuration>
+                            <instructions>
+                                <!--OSGi imports-->
+                                <!-- All backend dependencies should be specified here -->
+                                <Import-Package>
```

Comment:
Looks like Import-Package is done twice. Seems this section is not needed as it is done with properties at line 104